README.md

```diff
@@ -19,6 +19,15 @@ Marquez is an open source **metadata service** for the **collection**, **aggrega
 
 Marquez is an [LF AI & Data Foundation](https://lfaidata.foundation/projects/marquez) incubation project under active development and we'd love your help!
 
+## Code of Conduct
```

Comment:
With the contributor covenant badge added to our _Badges_ section, this section can be removed.